packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -230,16 +234,16 @@ def convert_from_enum(cls, v):
             return NodeState(currentStatus=running_state_value)
         return v
 
-    class Config:
-        extra = Extra.forbid
-
+    def _patch_json_schema_extra(self, schema: dict) -> None:
         # NOTE: exporting without this trick does not make runHash as nullable.
         # It is a Pydantic issue see https://github.com/samuelcolvin/pydantic/issues/1270
```

Comment:
so this NOTE goes with the previous file contents. is this now ok? if yes then the NOTE should probably go away